.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules
 dist/*
 !dist/wikibase.termbox.main.js
 !dist/wikibase.termbox.main.css
+!dist/img/*
```

Comment:
We would want this (if it were a thing) to be ignored (L3 does that already) - in wikibase we can use the `assets` folder.

Reply:
img we be there if you make the build with the example branch. In this pull request there is nothing which links to this asset, therefore it will not compiled.
I agree it is not the best way to have assets at least twice, but so far I have no solution to provide a better way then this.